main.py

```diff
@@ -246,6 +253,48 @@ def generate_json():
 
     return {"room_days": room_days_info}
 
+@app.route('/json/<day>/<room>')
```

Comment:
@ldang: did you want it specifically `/<day>/<room>/json` because you want to directly append to the url and copy json from the browser?

Reply:
I realized later that this way made more sense.